docs/services/education-consultant.md

```diff
@@ -0,0 +1,31 @@
+# Education Consultant
+
+The Education Consultant has experience with designing curriculum and creating positive learning spaces for data science education.
+
```

Comment:
We can include the learning zones diagram (my favorite ;) ) as a photo here:
![image](https://user-images.githubusercontent.com/1154958/213310938-90671253-936a-408f-9e11-42146e18487b.png)

Reply:
Here's another photo that might fit, from one of my earlier teaching experiences. I think it captures the terror of being a tutorial lead quite nicely. And we can help with that!
![Naomi teaching](https://nlalterman.com/static/in_class_2.jpg)